content/api/message-events.apib

```diff
@@ -4,87 +4,158 @@ description: Query detailed message event data for further analysis and processi
+The Message Events API allows searching for recent events, and supports various types of result filtering. Available events include message status – injection, delivery, bounce – as well as recipient engagement – clicks and opens. You can fetch all event types or only specific ones, such as bounces, deliveries, or clicks. You can also filter by date range, campaign, or just about any other field. It provides access to the same event data that gets delivered to you through [webhooks](/api/webhooks).
 
-You can get all event types or only specific ones, such as bounces, deliveries, or clicks. You can filter by date range, campaign, or just about any other field. Event data is retained for 10 days and is generally available within 1 minute. Of course, aggregate reporting data is available via our metrics endpoint or the UI reports for much longer.
+### Data retention
 
-## Data retention
+Message event data is retained for 10 days and is generally available within 1 minute. Aggregate reporting data is available through [metrics](/api/metrics) or the app for up to 6 months.
 
-Message Events API data is retained for 10 days.
+___
 
-## Copyrights
-__**The SparkPost message events API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)**__
+__**The SparkPost Message Events API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)**__
 
-## Events Documentation [/message-events/events/documentation]
 
-### Documentation [GET]
+### Search for Message Events [GET /message-events{?recipients,friendly_froms,events,bounce_classes,reason,subaccounts,transmission_ids,campaign_ids,template_ids,message_ids,timezone,to,from,page,per_page,delimiter}]
 
-List descriptions of the event fields that could be included in a response from the Message Events search endpoint. Fields will vary by event type.
+Returns a list of message events that matched the filtered search. The response is sorted by descending timestamp.
+
++ Parameters
+    + recipients (optional, list, `recipient@example.com`) ... Delimited list of recipients to search.
+    + friendly_froms (optional, list) ... Delimited list of friendly from emails to search. 
+    + events (optional, list) ... Delimited list of event types to search. Defaults to all event types.
+    + bounce_classes (optional, number) ... Delimited list of bounce classification codes to search. See [Bounce Classification Codes](https://www.sparkpost.com/docs/deliverability/bounce-classification-codes/).
+    + reason (optional, string) ... Bounce/failure/rejection reason that will be matched using a wildcard (e.g., %reason%).
```

Comment:
Maybe just me, but `%reason%` doesn't mean anything to me?

Reply:
Gonna leave it for now, but will clarify with the team later.